### PR TITLE
Tutorials update

### PR DESCRIPTION
**Pull request by mtharrison**
I only had time to update these for now. At your discretion to either merge or leave open and I can work on the rest next week.

### CODE REVIEW

File: lib/tutorials/caching.md [hunk 1]
```diff
@@ -154,7 +157,13 @@ var add = function (a, b, next) {
     return next(null, Number(a) + Number(b));
 };
 
-server.method('sum', add, { cache: { cache: 'mongoCache', expiresIn: 30 * 1000 } });
+server.method('sum', add, {
```
Comment (gergoerdosi) [inline lib/tutorials/caching.md]: This example should have 4 spaces for consistency.

File: lib/tutorials/serving-files.md [hunk 2]
```diff
@@ -36,12 +53,23 @@ var server = new Hapi.Server({
...
+
+    server.route({
+        method: 'GET',
+        path: '/picture.jpg',
+        handler: function (request, reply) {
+            reply.file('path/to/picture.jpg');
+        }
+    });
+
+    server.start(function () {
```
Comment (AdriVanHoudt) [inline lib/tutorials/serving-files.md]: handle the new err it can give back?

File: lib/tutorials/serving-files.md [hunk 3]
```diff
@@ -96,7 +124,7 @@ server.route({
 
 ## Directory handler
 
-In addition to the `file` handler, there is also a `directory` handler that allows you to specify one route to serve multiple files. In order to use it, you must specify a path with a parameter. The name of the parameter does not matter, however. You can use the asterisk extension on the parameter to restrict file depth as well. The most basic usage of the directory handler looks like:
+In addition to the `file` handler, inert also adds a `directory` handler that allows you to specify one route to serve multiple files. In order to use it, you must specify a path with a parameter. The name of the parameter does not matter, however. You can use the asterisk extension on the parameter to restrict file depth as well. The most basic usage of the directory handler looks like:
```
Comment (AdriVanHoudt) [inline lib/tutorials/serving-files.md]: s/inert/Inert (with link or in bold or something)

File: lib/tutorials/serving-files.md [hunk 4]
```diff
@@ -1,18 +1,35 @@
...
 
 First, to use the reply method:
 
 ```javascript
-server.route({
-    method: 'GET',
-    path: '/picture.jpg',
-    handler: function (request, reply) {
-        reply.file('/path/to/picture.jpg');
+server.register(Inert, function (err) {
```
Comment (gergoerdosi) [inline lib/tutorials/serving-files.md]: I think you should mention what `Inert` refers to, or just use `require('inert')` in `server.register()`.

**Comment by AdriVanHoudt** [comment]
Nice work!

File: lib/tutorials/serving-files.md (hunk 3 above)
Comment (mtharrison) [inline lib/tutorials/serving-files.md]: As this isn't a reference to the variable, I'm not sure it should be uppercase I. The lib isn't: https://github.com/hapijs/inert

Comment (AdriVanHoudt) [inline lib/tutorials/serving-files.md]: ok fair point

Comment (mtharrison) [inline lib/tutorials/serving-files.md]: There's already a link to inert right at the top. Do you think it's necessary to link every instance?

Comment (AdriVanHoudt) [inline lib/tutorials/serving-files.md]: in that case nope, just did a quick scan sorry :P

File: lib/tutorials/serving-files.md (hunk 4 above)
Comment (mtharrison) [inline lib/tutorials/serving-files.md]: :+1:

File: lib/tutorials/serving-files.md (hunk 3 above)
Comment (mtharrison) [inline lib/tutorials/serving-files.md]: No prob :)

**Comment by hueniverse** [comment]
We can merge as each tutorial is done, just let me know when all the comments are addressed.

**Comment by mtharrison** [comment]
They've all been addressed now.

**Comment by nlf** [comment]
checking it out right now

File: lib/tutorials/caching.md [hunk 5]
```diff
@@ -71,7 +73,7 @@ Catbox has two interfaces; client and policy.
 
 [Client](https://github.com/hapijs/catbox#client) is a low-level interface that allows you set/get key-value pairs. It is initialized with one of the available adapters: ([Memory](https://github.com/hapijs/catbox-memory), [Redis](https://github.com/hapijs/catbox-redis), [mongoDB](https://github.com/hapijs/catbox-mongodb), [Memcached](https://github.com/hapijs/catbox-memcached), or [Riak](https://github.com/DanielBarnes/catbox-riak)).
 
-hapi always initialize one default [client](https://github.com/hapijs/catbox#client) with [memory](https://github.com/hapijs/catbox-memory) adapter. Lets see how we can define more clients.
+hapi always initialize one default [client](https://github.com/hapijs/catbox#client) with [memory](https://github.com/hapijs/catbox-memory) adapter. Let's see how we can define more clients.
```
Comment (nlf) [inline lib/tutorials/caching.md]: my own typo that i just noticed, but this should say `initializes` with an s

File: lib/tutorials/caching.md (hunk 1 above)
Comment (nlf) [inline lib/tutorials/caching.md]: +1

**Comment by nlf** [comment]
nice, thanks for the effort on this @mtharrison. looks really good. i'm gonna go ahead and merge and i'll fix that one typo out of band.

**Comment by AdriVanHoudt** [comment]
nice job! :+1:

**Pull request merged**